xrpl-wasm-stdlib/src/core/types/account_id.rs

```diff
@@ -25,3 +32,95 @@ impl From<[u8; ACCOUNT_ID_SIZE]> for AccountID {
         AccountID(value)
     }
 }
+
+/// Implementation of `FieldGetter` for XRPL account identifiers.
+///
+/// This implementation handles 20-byte account ID fields in XRPL ledger objects.
+/// Account IDs uniquely identify accounts on the XRPL network and are derived
+/// from public keys using cryptographic hashing.
+///
+/// # Buffer Management
+///
+/// Uses a 20-byte buffer (ACCOUNT_ID_SIZE) and validates that exactly 20 bytes
+/// are returned from the host function. The buffer is converted to an AccountID
+/// using the `From<[u8; 20]>` implementation.
+impl FieldGetter for AccountID {
```

Comment:
The naming of the trait `FieldGetter` is of course prior art that isn't currently changing in this PR. However, looking here, it seems like we should rename `FieldGetter` to be `LedgerObjectFieldGetter`, both for clarity and to align with the pattern below where we name the other trait `CurrentTxFieldGetter`.

Any objections?

Reply:
Agreed, but can that be in a fast-follow separate PR?

Reply:
Sounds good - want to create an issue for tracking if it doesn't exist?

Reply:
Easier to just create the PR: #109 (base is this branch)